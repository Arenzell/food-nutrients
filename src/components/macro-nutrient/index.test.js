import React from 'react'
import MacroNutrient from './index'
import renderer from 'react-test-renderer'

it('renders correctly with data', () => {
  const tree = renderer
    .create(<MacroNutrient name={'Example Macro'} amount={1000} units={'mg'} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders correctly without data', () => {
  const tree = renderer.create(<MacroNutrient name={'Example Macro'} />).toJSON()
  expect(tree).toMatchSnapshot()
})
