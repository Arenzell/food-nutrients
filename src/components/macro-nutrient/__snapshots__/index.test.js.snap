// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with data 1`] = `
<div
  className="macroNutrient"
>
  <div>
    Example Macro
  </div>
  <div>
    1000
     
    mg
  </div>
</div>
`;

exports[`renders correctly without data 1`] = `
<div
  className="macroNutrient"
>
  <div>
    Example Macro
  </div>
  <div>
    0
     
    
  </div>
</div>
`;
